sdk/identity/identity/test/internal/node/clientSecretCredential.spec.ts

```diff
@@ -92,7 +92,11 @@ describe("ClientSecretCredential (internal)", function () {
     assert.equal(doGetTokenSpy.callCount, 1);
   });
 
-  it("Authenticates with tenantId on getToken", async function () {
+  it("Authenticates with tenantId on getToken", async function (this: Context) {
+    // The live environment isn't ready for this test
+    if (isLiveMode()) {
+      this.skip();
+    }
```

Comment:
This works locally in all test modes. We’ll investigate outside of this PR: https://github.com/Azure/azure-sdk-for-js/issues/21055